### PR TITLE
Goldgrubs now run away from mechas

### PR DESCRIPTION
**Pull request by Nicaragua1**
Goldgurbs werent running away from mechas making them easy to be killed with a RIPLEY or any MECH
In fact they were aggro against mechas which made no sense since they do no damage to them.

-Tested: they dont run away from empty mechas.-

:cl:
Fix: Golgrubs now run away from mechas being piloted
/:cl:

### CODE REVIEW

**Comment by Aurorablade** [comment]
should be bugfix:<comment here> for your CL as per template.

File: code/modules/mob/living/simple_animal/hostile/mining/goldgrub.dm [hunk 1]
```diff
@@ -42,7 +42,7 @@
 	if(target != null)
 		if(istype(target, /obj/item/stack/ore) && loot.len < 10)
 			visible_message("<span class='notice'>The [name] looks at [target.name] with hungry eyes.</span>")
-		else if(isliving(target))
+		else if(isliving(target) || istype(target, /obj/mecha))
```
Comment (Aurorablade) [inline code/modules/mob/living/simple_animal/hostile/mining/goldgrub.dm]: else if(isliving(target) || ismecha(target))

Comment (Nicaragua1) [reply code/modules/mob/living/simple_animal/hostile/mining/goldgrub.dm]: That's much better thanks

**Comment by Aurorablade** [comment]
Wait are you using the web editor?

**Comment by Aurorablade** [comment]
Okay won't try to complain as it was like a one line thing and the first commit did do that. Typically alot of us don't like the web editor being used. Just PLEASE make sure to compile and test it and report it works.

**Comment by Nicaragua1** [comment]
> Wait are you using the web editor?
no

**Comment by Aurorablade** [comment]
yeah its just how the one commit was titled.

**Comment by Shadow-Quill** [comment]
Make sure you add "fix:" to your changelog

**Comment by KasparoVy** [comment]
Does this also make sure the mech is beigg piloted so as to avoid empty inactive mechs from freaking them out?

**Comment by Nicaragua1** [comment]
> Does this also make sure the mech is beigg piloted so as to avoid empty inactive mechs from freaking them out?

yep just like the description says. They dont run away from empty mechs.

**Pull request merged**